app/DomainValueObjects/Location/Location.php

```diff
@@ -3,31 +3,35 @@
 namespace App\DomainValueObjects\Location;
 
 use App\DomainValueObjects\UUIDGenerator\UUID;
+use App\DomainValueObjects\Location\GeoLocation;
+use App\DomainValueObjects\Location\Address;
 
-//Exceptions
-//UUID
 use App\Exceptions\UUIDExceptions\GenerateUUID5Failed;
 use App\Exceptions\LocationExceptions\LocationNotDefined;
-
-
+use App\Exceptions\LocationExceptions\GeoLocationNotDefined;
+use App\Exceptions\LocationExceptions\AddressNotDefined;
 
 class Location
 {
-    private $uuid;
-    private $location = null;
+    private $uuid = null;
+    private $geoLocation = null;
+    private $address = null;
 
     public function __construct(
```

Comment:
i think we need two constructors one with address as apart of the constructor (like this) and one with out for the purpose of: 
1. Using address when an admin is adding an address to an organization. 
2. Not using an address for clocking in / clocking out

Reply:
Maybe we can use a custom comparator ? when we clockin/clockout we receive coordinates which we make a new location object. then we compare the location object to user's designated location.